Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+#source 'https://rubygems.org'
```

Comment:
Is this on purpose?  Won't this breaks something?

Reply:
Nope, it was a stupid typo. Thanks for the catch!